mafia.py

```diff
@@ -11,7 +12,8 @@
 id = 0
 nPlayers = 0
 roles = []
-ip2role_index_name = {}
+#ip2role_index_name = {}
+ip2players = {}
```

Comment:
> If you have time, I think it would be great if we make a class for `Role` too.
I couldn't agree more. Actually this is what I am currently working on.